samples/knative/quickstart/platformsetup.sh

```diff
@@ -114,12 +115,14 @@ fi
 ### Note: the preceding steps perform domain install; following steps perform Iter8 install
 
 # Step 5: Install Iter8
-echo "Installing Iter8"
-curl -s https://raw.githubusercontent.com/iter8-tools/iter8-install/main/install.sh | bash
+echo "Installing Iter8 with Knative support"
+curl -s https://raw.githubusercontent.com/${GIT_ACCOUNT}/iter8-install/${TAG}/install.sh | bash
```

Comment:
${TAG} needs to be replaced with  `main`

Reply:
@sriumcp I don't understand why this is the case. Why do we not want to install a particular version that the sample was written/tested against?

Reply:
What we want to ensure is that a specific version of Iter8 corresponds to a specific version of `iter8-install` (and `iter8ctl`).

Once you pin the version of `iter8-install`, by extension, you also pin the version of `handler`, `etc3` and `iter8-analytics`.

If tutorial points to a different version of Iter8, then we are will pretty soon be staring at an unmanageable mess in terms of dependencies.

Reply:
Re: testing, we do want to test the samples against the `iter8-install` version that install.md points to.

If the tutorials break for the version installed by `install.md`, we have a problem, and shouldn't be merging in the first place.

Reply:
ok. I think I follow the expected behavior. I have undone most of the changes in this script.